FirebasePerformance/Tests/Unit/Instruments/FPRNSURLSessionInstrumentTest.m

```diff
@@ -381,8 +381,12 @@ - (void)testDelegateURLSessionTaskDidSendBodyDataTotalBytesSentTotalBytesExpecte
   NSURL *URL = [self.testServer.serverURL URLByAppendingPathComponent:@"testUpload"];
   NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:URL];
   request.HTTPMethod = @"POST";
-  NSURL *fileURL = [[NSBundle bundleForClass:[self class]] URLForResource:@"smallDownloadFile"
-                                                            withExtension:@""];
+#if SWIFT_PACKAGE
```

Comment:
Consider moving getting the bundle to a single utility function

Reply:
Sounds good! Done!